src/App.jsx

```diff
@@ -14,70 +15,54 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
For the default value, it is better to take the value directly from the `goods` array.
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```

Reply:
maybe it's better, but according to the task: **Keep selected Good in state (Jam is default)**.
and I think that if the list of goods changes, then Jam may not be [8]. and it will be not according task

Reply:
It's hard to say - but overall I would agree.
In the scope of this very task it is ok to keep it